projects/noloan/app/src/main/java/hasadna/noloan/SpamRecyclerAdapter.java

```diff
@@ -18,26 +18,26 @@
 public class SpamRecyclerAdapter
     extends RecyclerView.Adapter<SpamRecyclerAdapter.RecyclerViewHolder> {
 
-  SpamHolder spam;
+  DBMessagesHolder spam;
 
   public SpamRecyclerAdapter() {
-    spam = SpamHolder.getInstance();
+    spam = DBMessagesHolder.getInstance();
```

Comment:
So this is no longer just `spam` but also `suggestions`. How about we rename to `messages` or `dbMessages`?

Reply:
dbMessages  :+1: